GraphicsTemplate/GraphicsTemplate/GraphicsTemplate.csproj

```diff
@@ -99,4 +99,4 @@
   <Target Name="AfterBuild">
   </Target>
   -->
-</Project>
+</Project>
```

Comment:
@weterbogov how about fixing end of file? :)

Reply:
@thatglance I think it is not neccessary fix auto created files, because of this files will be rewrited more than 1 times